lib/filewatch/observing_tail.rb

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+require_relative 'bootstrap'
+require_relative 'tail_handlers/processor'
+require "logstash/util/loggable"
+
+module FileWatch
+  class ObservingTail
+    include LogStash::Util::Loggable
+    include ObservingBase
+
+    def build_specific_processor(settings)
```

Comment:
as discussed, I think it would be useful to document somehow the interface for the `ObservingXxx` classes

Reply:
👍